### ISSUE TITLE
Add "awaiting ticket" flag to Flagged Content view and all facility content types & remove "Awaiting CSV" flag

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#13624]
## Description 
In Drupal, when the Facilities migration runs nightly, it adds flags to Facility nodes based on certain criteria. In this ticket we want to add a flag, and remove a flag. 

### Add: `Awaiting Github ticket`
Create a new flag for "Awaiting Github ticket".
This should be added to any newly flagged facility that was flagged for any other reason (New facility, Removed from source, Changed name). 
This will track the fact that Helpdesk will need to create a Github ticket to track any steps required to mitigate the other flags. 

We will also add the reference to this flag to any [runbooks](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/flagged-facilities.md) for New / Changed / Removed, to indicate the flag should be removed once ticket is created. 

### Remove `Awaiting CSV`
Lighthouse is [making a change](https://dsva.slack.com/archives/C02730UEZPS/p1740432213877859) that will eliminate their URL CSV file. Drupal will now be source of truth for facility URLs for modernized facilities. We've updated runbooks for this in #20676. Now: we will remove the flag itself from Drupal. 

This should include removing the flag from any nodes where it appears.

## ACs

- [ ] I expect that the `Awaiting Github ticket` is added to any facility flagged during the nightly facility migration, for: 
    - New facility
    - Removed from source
    - Changed name 
- [ ] I expect `Awaiting Github ticket` flag is visible on [Flagged Content view](https://prod.cms.va.gov/admin/content/facilities/flagged)
- [ ] I expect that removing the `Awaiting Github ticket` flag from a facility will create a revision log message on the facility node.
- [ ] I expect that `Awaiting CSV` flag is removed from Drupal 
    - [ ] no longer appears in Flagged Content view
    - [ ] Is not applied automatically by any process during Facilities migration
    - [ ] is not applied to any Facility node
- [ ] I expect all runbooks for New, Removed, or Changed name facilities are updated to add instructions re: removing the `Awaiting Github ticket` flag from the Facility node, when tickets are created

**Comment by jilladams** [comment]
@swirtSJW can you add your thoughts re: not adding Github actions here?

**Comment by jilladams** [comment]
From 5/18 refinement: concerns around bulk actions are: 
* Can't think of a case where we would clear out a mass of flags without adding specific data to each.
* Bulk Ops is handy but skips processes, such as flagging / de-flagging node will save messages to Revision log. Bulk Ops would not handle this the same way.

**Comment by kmariepat-cityfriends** [comment]
@kmariepat-cityfriends to give a @jilladams a heads up when this is prioritized in a sprint

**Comment by swirtSJW** [comment]
Implementation ~details~ thoughts:   I mentioned in refinement that the auto flagging would need to be added in two locations.
1. the ones that happen on facility save.  (new facility, name change)
2.  the one that happens after the entire migration is run. (removed from source)

It occurs to me we might be able to catch all cases in flag entity save (hook/event).   Then it could all be kept in one spot and act on adding the "awaiting ticket" flag when ever adding new, changed, removed from source.

**Comment by Agile6MSkinner** [comment]
@jilladams Is this still something we want to do?

**Comment by jilladams** [comment]
Definitely. It would be massively helpful for the Flagged Facilities workflow, if we could get it done.
